templates/cicd/pipeline_cfn.yml

```diff
@@ -93,10 +93,46 @@ Resources:
         Type: CODEPIPELINE
         BuildSpec: ecs-project/buildspec.yml
       TimeoutInMinutes: 60
+  IntegTestRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub ${AWS::StackName}-IntegTestRole
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          -
+            Effect: Allow
+            Principal:
+              Service:
+                - codebuild.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /{{$length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}
+  IntegTestProject{{$stage.Name}}:
```

Comment:
Is it possible that the stage name contains a dash? if so we might want to do something similar to:
https://github.com/aws/amazon-ecs-cli-v2/blob/e19329b06fcf1c77d048fa7d85f803d0fa3bcb4f/internal/pkg/deploy/cloudformation/stack/template_functions.go#L12-L22

Reply:
lol sure :)

Reply:
Oh actually the stage name can contain dash. Take a look at the screenshot in the "conversation" tab